docs/go/how-to-continue-as-new-in-go.md

```diff
@@ -18,3 +18,9 @@ func SimpleWorkflow(ctx workflow.Context, value string) error {
 ```
 
 To check whether a Workflow Execution was spawned as a result of Continue-As-New, you can check if `workflow.GetInfo(ctx).ContinuedExecutionRunID` is not nil.
+
+## Notes
```

Comment:
```suggestion
**Notes**
```
Dropping the heading to ensure that the topic renders as expected in the app dev guide.